### ISSUE TITLE
nan occurred when training Alexnet

### ISSUE THREAD

**Issue by jwook1004** [tensorpack/tensorpack#796]
Issue: Unexpected Problems / Potential Bugs

1. What you did:
I just git-clone this repository and ran training for Alexnet using the python script in "https://github.com/tensorpack/tensorpack/blob/master/examples/ImageNetModels/alexnet.py".
Command: 
`python3 alexnet.py --gpu 0,1 --data /home/choij/Data/ILSVRC2012 --batch 64`

2. What you observed, e.g. the entire log:
> [0618 07:42:45 @base.py:242] Start Epoch 1 ...
> [0618 07:42:45 @input_source.py:513] Pre-filling StagingArea ...
> [0618 07:42:45 @input_source.py:517] 1 element was put into StagingArea.
> 100%|#####################################################################################################################################################################|10009/10009[13:44<00:00,12.14it/s]
> [0618 07:56:29 @base.py:252] Epoch 1 (global_step 10009) finished, time:13 minutes 44 seconds.
> [0618 07:56:29 @graph.py:73] Running Op sync_variables/sync_variables_from_main_tower ...
> [0618 07:56:30 @saver.py:84] Model saved to train_log/AlexNet/model-10009.
> [0618 07:56:30 @misc.py:98] Estimated Time Left: 22 hours 41 minutes 38 seconds
> 100%|#########################################################################################################################################################################|782/782[00:15<00:00,49.52it/s]
> [0618 07:56:46 @monitor.py:440] DataParallelInferenceRunner/QueueInput/queue_size: 50
> [0618 07:56:46 @monitor.py:440] GPUUtil/0: 93.674
> [0618 07:56:46 @monitor.py:440] GPUUtil/1: 94.415
> [0618 07:56:46 @monitor.py:440] QueueInput/queue_size: 50
> [0618 07:56:46 @monitor.py:440] l2_regularize_loss: nan
> [0618 07:56:46 @monitor.py:440] learning_rate: 0.1
> [0618 07:56:46 @monitor.py:440] train-error-top1: 1
> [0618 07:56:46 @monitor.py:440] train-error-top5: 1
> [0618 07:56:46 @monitor.py:440] val-error-top1: 1
> [0618 07:56:46 @monitor.py:440] val-error-top5: 1
> [0618 07:56:46 @monitor.py:440] xentropy-loss: nan
> [0618 07:56:46 @group.py:44] Callbacks took 16.977 sec in total. DataParallelInferenceRunner: 16 seconds

3. What you expected, if not obvious.
As can be seen above, I got "[nan](https://en.wikipedia.org/wiki/NaN)" for l2_regularize_loss and xentropy-loss. I expect valid values for them. Note that I didn't modify any code in the script (i.e., alexnet.py) or other Tensorpack source code.

4. Your environment:
	+ Python version.: Python 3.6.5
	+ TF version: v1.7.0-3-g024aecf414 1.7.0
	+ Tensorpack version: 0.8.6

**Comment by ppwwyyxx** [comment]
Thanks for reporting! Forgot to update the initial learning rate when I refactor the code. Should be fixed now.
